src/Xamarin.Android.Build.Tasks/Mono.Android/ApplicationAttribute.Partial.cs

```diff
@@ -57,6 +57,12 @@ static partial void AddManualMapping ()
 
 		public static ApplicationAttribute FromCustomAttributeProvider (ICustomAttributeProvider provider, TypeDefinitionCache cache)
 		{
+			// `provider` might be null in situations when application configuration is broken, and it surfaces in a number of
```

Comment:
I'd like to know more about this scenario? How is `provider` null when the application config is broken?

Reply:
This happened in the XA0119 test (which is, itself, a broken configuration scenario) where there were no assemblies to load and the code in ManifestDocument would iterate over the list of assemblies like so:

```c#
foreach (var assemblyPath in Assemblies) {
  var assembly = Resolver.GetAssembly (assemblyPath);
  if (ApplicationAttribute.FromCustomAttributeProvider (assembly, cache) is ApplicationAttribute a) {
    assemblyAttr.Add (a);
  }
}
```

Each `assembly` is an instance of `ICustomAttributeProvider`, but `Resolver.GetAssembly` would return `null` since the assembly couldn't be found.